service/src/main/resources/HTMLTemplate.html

```diff
@@ -798,27 +800,31 @@ <h3 class="heading heading--two">Child {{ counter }}</h3>
             <dt class="data-list__title">Child's living situation</dt>
             <dd class="data-list__definition">
               {% if child.value.livingSituation is not empty %}
-                <span class="u-block">{{child.value.livingSituation}}</span>
-                {% if child.value.address is not empty %}
-                  {% if child.value.address.AddressLine1 is not empty %}
-                    <span class="u-comma-before">{{ child.value.address.AddressLine1 }}</span>
-                  {% endif %}
-                  {% if child.value.address.AddressLine2 is not empty %}
-                    <span class="u-comma-before">{{ child.value.address.AddressLine2 }}</span>
-                  {% endif %}
-                  {% if child.value.address.AddressLine3 is not empty %}
-                    <span class="u-comma-before">{{ child.value.address.AddressLine3 }}</span>
+                <span class="u-block">{{ child.value.livingSituation }}</span>
+                  {% if child.value.address is not empty %}
+                    {% if child.value.detailsHidden is not empty and child.value.detailsHidden == "Yes"%}
```

Comment:
Since you make other changes it would be good to add whitespace before closing `}`